sdk/rs/src/client.rs

```diff
@@ -641,7 +641,7 @@ mod utils {
                 AccountMeta::new(self.multisig, false),
                 AccountMeta::new_readonly(self.member, true),
                 AccountMeta::new(self.proposal, false),
-                AccountMeta::new_readonly(self.transaction, false),
+                AccountMeta::new(self.transaction, false),
```

Comment:
I don't think we should change this for 0.1.0. That version is meant to be working with the program version 2.0.0 (currently on staging)

we have two options:
- if there's no time pressure for releasing the CLI before the production program upgrade (probably happens early next week), then just use the method as it is and test with the staging program for now
- if we need to release before the upgrade- just copy/paste the implementation if into_accounts_metas into the cli crate for now and later refactor into using the sdk implementation